PotreeConverter/CMakeLists.txt

```
@@ -30,12 +31,13 @@ SOURCE_GROUP("Source Files\\io" FILES
 	src/LASPointReader.cpp
 	src/LASPointWriter.cpp
 	src/PTXPointReader.cpp
+	src/FlatBufferReader.cpp
 )
 
 SOURCE_GROUP("lib\\rapidjson" FILES ${lib_rapidjson_files})
 SOURCE_GROUP("lib\\arguments" FILES ${lib_argument_files})
 
-add_executable(PotreeConverter ${files} ${lib_rapidjson_files} ${lib_argument_files})
+add_executable(PotreeConverter ${files} ${lib_rapidjson_files} ${lib_argument_files} include/FlatBufferReader.hpp src/FlatBufferReader.cpp)
```

```suggestion
add_executable(PotreeConverter ${files} ${lib_rapidjson_files} ${lib_argument_files})
```
I don't think the explicit includes are needed here since they get included by `${files}`